fossunited/chapters/web_template/foss_club___criteria_for_lead/foss_club___criteria_for_lead.html

```diff
@@ -123,4 +123,4 @@ <h5>{{ criteria_title }}</h5>
       <img src="assets/fossunited/images/illustrations/criteria_board.svg" alt="" />
     </div>
   </div>
-</div>
+</div>
```

Comment:
```suggestion
</div>

```

re-added newline at end of file. this is one of the conventions that we follow with source code